drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -933,7 +937,7 @@ def can_see_client_details?(user)
     end
 
     PERMANENT_DESTINATIONS = HudSpmReport::Generators::Fy2023::MeasureSeven::PERMANENT_DESTINATIONS
-    PERMANENT_DESTINATIONS_OR_STAYER = (PERMANENT_DESTINATIONS + [0]).freeze
+    # PERMANENT_DESTINATIONS_OR_STAYER = (PERMANENT_DESTINATIONS + [0]).freeze
```

Comment:
it sure seems like destination is not set to 0 for stayers, I've tried to adapt the spm_fields config to account for what I assume is a change in behavior in the spm implementation

Reply:
We have a comment `name: :moved_in_destination, # NOTE: destination 0 == stayer in the SPM` on `cells: [['7b.2', 'C2']],`.  Before we remove this I'd like to confirm this is no longer true.  
Looking at some data in production I'm not seeing any SPM Enrollments with `destination = 0` so probably this change should be:
```
PERMANENT_DESTINATIONS_OR_STAYER = (PERMANENT_DESTINATIONS + [nil]).freeze
```

Reply:
seems reasonable, I'll try it, thanks. I looked around to see if we had been setting this to 0 in older SPMs but couldn't find any clues.